app/src/main/java/com/tnco/runar/ui/fragment/HasVisibleNavBar.kt

```diff
@@ -0,0 +1,3 @@
+package com.tnco.runar.ui.fragment
+
+interface HasVisibleNavBar
```

Comment:
added a marker interface to identify fragments that should display the navigation bar